defaults-o2-dataflow.sh

```diff
@@ -30,6 +30,8 @@ disable:
   - O2sim
   - O2-full-system-test
 overrides:
+  protobuf:
```

Comment:
@teo I think this needs to be removed before?

Reply:
Sounds right, is there any impediment to doing so?

Reply:
@ktf could you confirm that we can go ahead with this suggestion ?

Reply:
Not until you are happy with dropping GCC 7.3, since 5.15.8 will not compile there.

Reply:
Alternative @ironMann should find a workaround like you did to use 3.15.8 in DD & the rest of the EPN stack.

Reply:
I will unbump the minimum version of Protobuf for OCC back to v3.14.0 so we can still build the FLP Suite with defaults `o2-dataflow`.

Reply:
Fix here https://github.com/alisw/alidist/pull/3014